components/dataPrivacy/HeroSection.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import styled         from "styled-components";
+import { breakpoint } from "../../frontend-config";
+
+import { Heading1, TextBody } from "../shared/styles";
+
+const Section = styled.div`
+  text-align: center;
+  background: ${props => props.theme.colors.backgroundPrimary};
+  padding: 35px 75px 38px;
+
+  @media (min-width: ${breakpoint("lg")}) {
+    padding: 40px 75px 62px;
+  }
+`;
+
+const Title = styled(Heading1)`
+  font-size: 31px;
+  color: ${props => props.theme.colors.greenPrimary};
+  margin-bottom: 2px;
+
+  @media (min-width: ${breakpoint("lg")}) {
+    margin-bottom: 12px;
+    font-size: 53px;
+  }
+`;
+
+const HeroSection = () => {
```

Comment:
We have a 'PageHeader' component in `components/shared`. Instead of writing custom headers/heroes for each little page I believe it is preferred to use that and just pass in your page title.
So, please import 'PageHeader' to your page and this file would no longer be needed.